src/views/contributors.ts

```diff
@@ -13,62 +13,209 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { ManuscriptNodeView } from '@manuscripts/transform'
-import React from 'react'
+import { Capabilities, SecondaryButton } from '@manuscripts/style-guide'
+import { ContributorNode, isContributorNode } from '@manuscripts/transform'
 
-import { BaseNodeView } from './base_node_view'
+import { affiliationsKey } from '../plugins/affiliations'
+import { TrackableAttributes } from '../types'
+import BlockView from './block_view'
 import { createNodeView } from './creators'
 import { EditableBlockProps } from './editable_block'
 import ReactSubView from './ReactSubView'
 
 export interface ContributorsProps extends EditableBlockProps {
-  components: Record<string, React.ComponentType<unknown>>
-  subscribeStore: unknown
+  getCapabilities: () => Capabilities
+  openAuthorEditing: () => void
+  selectAuthorForEditing: (authorId: string) => void
 }
 
-interface AuthorsViewProps {
-  showAuthorEditButton: boolean
-  disableEditButton?: boolean
-  subscribe: (fn: (state: unknown, prev: unknown) => void) => () => void
-}
-
-export class ContributorsView<PropsType extends ContributorsProps>
-  extends BaseNodeView<PropsType>
-  implements ManuscriptNodeView
-{
-  private metadata: HTMLDivElement
-
+export class Contributors<
```

Comment:
Can we call this `ContributorsView` for consistency?

Reply:
Updated